tests/tools/cohort_tracking/test_cohort_tracking.py

```diff
@@ -0,0 +1,169 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import ehrapy as ep
+import ehrapy.tools.feature_ranking._rank_features_groups as _utils
```

Comment:
That's a weird import like this. Why `_utils`?

Reply:
I have absolutely no idea why this is here and just noticed this now hehe

Reply:
Done